tutorials/cloud-iot-logging/sample-device/src/client.ts

```diff
@@ -0,0 +1,105 @@
+/**
+ * Copyright 2018 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import mqtt = require('mqtt');
+import { TokenGenerator, SignAlgorithm } from "./token";
+import { setInterval } from 'timers';
+import { Observable } from 'rxjs';
+import 'rxjs/add/observable/fromEvent';
+// import {BehaviorSubject} from 'rxjs/BehaviorSubject';
```

Comment:
Maybe add a comment for why this line is commented out

Reply:
removed this and a few others